src/main/java/uk/gov/register/presentation/representations/NewRepresentationWriter.java

```diff
@@ -27,15 +30,21 @@ public final long getSize(View t, Class<?> type, Type genericType, Annotation[]
 
     @Override
     public boolean isWriteable(Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType) {
-        return NewEntryView.class.isAssignableFrom(type);
+        return NewEntryView.class.isAssignableFrom(type) ||
+                ItemView.class.isAssignableFrom(type);
     }
 
     @Override
     public void writeTo(View view, Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType, MultivaluedMap<String, Object> httpHeaders, OutputStream entityStream) throws IOException, WebApplicationException {
         if (view instanceof NewEntryView) {
             writeEntryTo(entityStream, requestContext.getRegister().getFields(), ((NewEntryView) view).getEntry());
         }
+        if (view instanceof ItemView) {
```

Comment:
Is it possible to implement `writeTo` method in such a way that it can convert the view object in any representation without knowing its type?